Apps/phazurevmmanagement/microsoftazurevmmanagement_connector.py

```diff
@@ -253,6 +254,20 @@ def __init__(self):
         self._access_token = None
         self._refresh_token = None
 
+    def _is_ip(self, input_ip_address):
+        """ Function that checks given address and returns True if address is valid IPv4 or IPV6 address.
+
+        :param input_ip_address: IP address
+        :return: status (success/failure)
+        """
+
+        try:
+            ipaddress.ip_address(input_ip_address)
+        except:
```

Comment:
```suggestion
        except ValueError:
```

also in general avoid bare excepts: https://www.flake8rules.com/rules/E722.html

Reply:
updated